auto_hpl/verifiction_config/auto_hpl_verify

```diff
@@ -0,0 +1,7 @@
+Required_Systems: intel,amd,arm
+Via_Zathras: No
+fields: 1:s,2:n,3:n,4:n,5:n,6:n;7:n
```

Comment:
Only using field numbering to identify the metrics isn't really readable or scalable..

What happens if we need to change the order of the CSV fields (for instance if Passmark adds/removes tests)?  Working with field numbering would end up being very annoying for any future maintenance.

Using field names probably would be better, since it'll be easier to read.

Reply:
Any change to the csv field will require updates to the test to start with.  If passmark adds or deletes a test, an updated results file is going to be required.  I am open to suggestions, but do not see an easy way to address your concern.